### PR TITLE
update readme (todo: cfg)

### PR DESCRIPTION
**Pull request by holiman**
This updates the Readme to contain the new code. 

It also changes some meta in the code (but this PR does not modfy the compiled code) I don't like that `revert` is a label, `revert()` is a macro, and, well, `revert` is also an opcode. So this PR disambiguates them

### CODE REVIEW

File: src/main.etk [hunk 1]
```diff
@@ -37,8 +37,8 @@
         0xfffffffffffffffffffffffffffffffffffffffe
 %end
 
-# revert sets up and then executes a revert instruction.
-%macro revert()
+# m_revert sets up and then executes a revert(0,0) operation.
+%macro m_revert()
```
Comment (lightclient) [inline src/main.etk:41]:
```suggestion
%macro do_revert()
```

File: src/main.etk [hunk 2]
```diff
@@ -116,9 +116,9 @@ loadroot:
         push0            # [offset, size]
         return           # []
 
-revert:
+lbl_revert:
```
Comment (lightclient) [inline src/main.etk:119]:
```suggestion
throw:
```

**Review by lightclient — COMMENTED** [review]
What do you think of these names?

**Comment by holiman** [comment]
> What do you think of these names?

I like it when labels are clearly labels, not opcodes or macros. So `%macro do_revert()` is pretty clear,  but, `push1 throw` is not obvious to me.  Why not `push1 lbl_throw`, so it's apparent that we're pushing a jump destination?

**Comment by lightclient** [comment]
The main issue is now one label is prefixed `lbl` and the others are not. So I don't think there should be inconsistency, regardless of approach.

Which part is non-obvious that `throw` is a label? Would a different non-`lbl` prefixed name be better? The language syntax (which may be a bit unclear :p) only allows constants, expression macros, and labels in the immediate position of a push opcode. So from the language POV, it is unambiguous.

File: README.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ $ disease --code 0x$(eas src/main.etk)
    0:   caller
    1:   push20 0xfffffffffffffffffffffffffffffffffffffffe
   16:   eq
-  17:   push1 0x44
+  17:   push1 0x4d # selector("swap26309536(address,address,uint256,uint256,uint256)")
```
Comment (lightclient) [inline README.md:34]:
```suggestion
  17:   push1 0x4d
```

**Review by lightclient — APPROVED** [review]

**Pull request merged**